core/config/docs/core.toml

```diff
@@ -470,6 +470,13 @@ MaxEncryptedSecretsSize = '26.40kb' # Default
 # MaxConfigSize is the maximum size of a config that can be fetched from the given config url.
 MaxConfigSize = '50.00kb' # Default
 
+[Workflows]
+[Workflows.Limits]
+# Global is the maximum number of workflows that can be registered globally.
+Global = 50 # Default
+# PerOwner is the maximum number of workflows that can be registered per owner.
+PerOwner = 5 # Default
```

Comment:
@agparadiso Can we set this to 200 Global and 200 PerOwner by default? This will be what we need for ST3